Assignment-1-Calculator.c

```diff
@@ -0,0 +1,130 @@
+#include<stdio.h>
+
+
+/* To check the error
+error - 1 for  invalid expression 
+error -2 for division by zero error */
+int error = 0;
+
+
+// The function is to check the priority of the operator according to the DMAS
+int priority_op(char operator){
+    if(operator == '*' || operator == '/')return 1;
+    if(operator == '+' || operator == '-')return 0;
+    return -1;
+}
+
+
+/* The fuction to  calculate the input */
+int calculate(char input[]){
+

```

Comment:
can have different function names to get more context and also can you split this multiple function doing specific task as this function is doing multiple tasks?